src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -148,18 +148,18 @@ export class ContextualMenu extends React.Component<IContextualMenuProps, IConte
 
   public render() {
     let { className,
-          items,
-          isBeakVisible,
-          labelElementId,
-          targetElement,
-          id,
-          targetPoint,
```

Comment:
The first contextual menu has a link at the bottom. Do you mean render a link and item outside of the contextual menu?